packages/function-entry/configure/src/feedback-mapper.js

```diff
@@ -0,0 +1,29 @@
+export const modelToFeedbackConfig = model => {
```

Comment:
we have something similar in `text-entry` - can we re-use that? could import like so: 
`import { feedbackMapper } from '@pie-lib/config-ui;`?

Reply:
At this point, the divergence between the two is quite substantial. There are different property names, different properties altogether and a slightly different structure. If perhaps a third component might need something similar further down the road we could maybe come up with a helper function that would help generate the structures that we need.

Reply:
👍